lib/omise/OmiseCapabilities.php

```diff
@@ -127,7 +126,7 @@ public function makeBackendFilterChargeAmount($amount)
         return function ($backend) use ($amount, $defMin, $defMax) {
             // temporary hack for now to correct min value for installments to fixed minimum (different to normal charge minimum)
             if ($backend->type === 'installment') {
-                $min = self::INSTALLMENT_MINIMUM;
+                $min = $this['limits']['installment_amount']['min'];
```

Comment:
* Is there a safe navigation operator we can use?
* Is this still a "temporary hack"?

Reply:
Sorry, I'm referring to the comment above:

>             // temporary hack for now to correct min value for installments to fixed minimum (different to normal charge minimum)

Can we remove the comment?

Reply:
Ah about that. I will remove it since this is not a temporary hack now.